File: WantList.Core/Manga.cs
using System;

namespace WantList.Core
{
    public class Manga
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string MangaUpdatesId { get; set; }
        public DateTime AddedDateTime { get; set; }
        public int WantRank { get; set; }
        public string MissingVolumes { get; set; }
        public bool Completed { get; set; }
        public bool Deleted { get; set; }
        public byte[] Image { get; set; }
    }
}

File: WantList.Angular/src/app/entities/manga.ts
export class Manga {
  id: number;
  name: string;
  mangaUpdatesId: string;
  addedDateTime: Date;
  wantRank: number;
  missingVolumes: string;
  completed: boolean;
  deleted: boolean;

  public static copyFrom( from: Manga, to: Manga ): void {
    Object.keys(from).forEach(key => to[key] = from[key]);
  }
}
